gef.py

```diff
@@ -1333,6 +1333,10 @@ def __str__(self) -> str:
                 f"last_remainder={self.last_remainder:#x}, next={self.n:#x}, next_free={self.nfree:#x}, "
                 f"system_mem={self.sysmem:#x})")
 
+    @property
+    def addr(self) -> str:
```

Comment:
Seems quite counter-intuitive to have a property called `addr` that is a string of an address in hex. Why not simply make it an `int` and format it as `#x`?

Reply:
resolved in 8454358